googlemock/test/gmock-more-actions_test.cc

```diff
@@ -325,7 +325,7 @@ TEST(InvokeTest, FunctionThatTakes10Arguments) {
 
 // Tests using Invoke() with functions with parameters declared as Unused.
 TEST(InvokeTest, FunctionWithUnusedParameters) {
-  Action<int(int, int, double, const string&)> a1 =
+  Action<int(int, int, double, const string)> a1 =
```

Comment:
Removed "const" from value parameters.

I can modify Travis scripts in a separate change that can be merged prior to this one.  It's possible, depending on which clang version Travis uses, that it may not experience this failure.  Regardless, this fixes undefined behaviour -- see the bug link up top.